src/Interpreters/Cache/FileCache.cpp

```diff
@@ -138,6 +139,11 @@ bool FileCache::isInitialized() const
     return is_initialized.load(std::memory_order_seq_cst);
 }
 
+bool FileCache::isFullyLoaded() const
```

Comment:
Changed to have `isInitialized()` return `is_fully_loaded`, while `assertInitialized()` still checks `is_initialized`

Reply:
Actually, probably we could do with only one variable, since the methods that call `assertInitialized()` should probably not be called until the cache is loaded